src/ws/cockpitwebservice.c

```diff
@@ -476,16 +261,6 @@ cockpit_web_service_dispose (GObject *object)
 
   cockpit_sockets_close (&self->sockets, NULL);
 
-  g_hash_table_iter_init (&iter, self->sessions.by_transport);
```

Comment:
That happens a couple lines above. Before the sockets gets closed. Or was the point that closes the transport should happen after the sockets?